libs/ngrx-toolkit/src/lib/with-storage-sync.spec.ts

```diff
@@ -1,14 +1,16 @@
 import { getState, patchState, signalStore, withState } from '@ngrx/signals';
-import { withStorageSync } from './with-storage-sync';
+import { withBigIntRehydration, withBigIntReplacement, withStorageSync } from './with-storage-sync';
```

Comment:
Would it be possible to rename `withBigIntRehydration` and `withBigIntReplacement` to `serialize` and `deserialize`. I think `bigint` will not be the last type where we require custom code.